tools/c7n_azure/c7n_azure/query.py

```diff
@@ -120,6 +137,35 @@ def get_query(self):
             self.manager.session_factory, self.manager)
 
 
+class TypeMeta(type):
+
```

Comment:
(minor) the new lines in these 3 classes seem to be inconsistent. It would probably be helpful to add comments on them too.